mllib/src/main/scala/org/apache/spark/mllib/evaluation/RegressionMetrics.scala

```diff
@@ -23,15 +23,23 @@ import org.apache.spark.Logging
 import org.apache.spark.mllib.linalg.Vectors
 import org.apache.spark.mllib.stat.{MultivariateStatisticalSummary, MultivariateOnlineSummarizer}
 import org.apache.spark.sql.DataFrame
-
 /**
  * Evaluator for regression.
  *
- * @param predictionAndObservations an RDD of (prediction, observation) pairs.
+ * @param predictionAndObservations an RDD of (prediction, observation) pairs,
+ * @param regThroughOrigin true if intercept is not included in linear regression model
```

Comment:
@dbtsai It may be nitpicking, but `RegressionMetrics` class can be used for any regression model and so `hasFitIntercept` doesn't make sense for all types of regressors (as mentioned in other comments). Someone evaluating a decision tree regression model might be confused by the parameter. I am not certain what is best, so I will defer to others' opinions.